doc/demo/alectryon.js

```diff
@@ -0,0 +1,172 @@
+var Alectryon;
```

Comment:
This is autogenerated code from Alectryon; not sure how I should add it. It's MIT-licensed, and we're simply using it rather than copying or modifying it. How does the licensing work in that case? Should I do some postprocessing to add a banner?

Reply:
Ah, just add it as is. Thanks.